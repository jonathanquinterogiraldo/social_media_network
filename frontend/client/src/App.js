import React, { Component } from 'react';
import {
  Switch,
  Route,
  //Redirect,
  //Link
} from 'react-router-dom';
import 'bootstrap/dist/css/bootstrap.min.css';
import Login from './components/Login'
import Home from './components/Home'
export class App extends Component {
  // constructor(props){
  //   super(props);  //  
  // }  

   render(){
    return (
      <div className="App">
        <Switch>
          <Route exact path='/'>
            <Login/>
          </Route>
          <Route exact path='/Home'>
            <Home/>
          </Route>
        </Switch>    
      </div>
    );
   } 
}

export default App;
