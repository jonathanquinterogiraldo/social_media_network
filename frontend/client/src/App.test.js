import React from 'react'
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import App from './App';
import { expect } from '@jest/globals'

describe('', () => {
  test('should mount <App /> correctly', async() => {
      
    const wrapper = shallow(<App />)
  
    expect(wrapper).toMatchSnapshot()    
  })  
})


