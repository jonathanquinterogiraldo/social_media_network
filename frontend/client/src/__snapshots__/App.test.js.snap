// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` should mount <App /> correctly 1`] = `
<div
  className="App"
>
  <Switch>
    <Route
      exact={true}
      path="/Login"
    >
      <Login />
    </Route>
    <ProtectedRoute
      path="/Home"
    >
      <Home />
    </ProtectedRoute>
    <Route
      exact={true}
      path="/Register"
    >
      <Register />
    </Route>
    <Route
      path="/"
    >
      <Login />
    </Route>
  </Switch>
</div>
`;

exports[`should mount <App /> correctly 1`] = `
<div
  className="App"
>
  <Switch>
    <Route
      exact={true}
      path="/Login"
    >
      <Login />
    </Route>
    <ProtectedRoute
      path="/Home"
    >
      <Home />
    </ProtectedRoute>
    <Route
      exact={true}
      path="/Register"
    >
      <Register />
    </Route>
    <Route
      path="/"
    >
      <Login />
    </Route>
  </Switch>
</div>
`;
